### PR TITLE
[13.0] Misc wishlist fix/imp

### PR DESCRIPTION
**Pull request by simahawk**
See atomic commits for details.

### CODE REVIEW

File: shopinvader_wishlist/services/wishlist.py [hunk 1]
```diff
@@ -494,12 +489,15 @@ def _replace_items(self, record, params):
...
-        return {
+        vals = {
             "product_set_id": record.id,
             "product_id": params["product_id"],
             "quantity": params.get("quantity") or 1,
-            "sequence": params.get("sequence") or 0,
         }
+        if "sequence" in params:
+            # Set sequence only if explicitly given
+            vals["sequence"] = params.get("sequence")
```
Comment (mmequignon) [inline shopinvader_wishlist/services/wishlist.py:499]: From my understanding, the validator will set 0 as a sequence by default, so sequence should always be in the params, therefore you should never enter in that block.
Am I missing something?

Comment (simahawk) [reply shopinvader_wishlist/services/wishlist.py:499]: this is why in the same change I've removed that default :wink:

**Comment by simahawk** [comment]
@mmequignon can you drop a review?

**Review by mmequignon — APPROVED** [review]

**Comment by simahawk** [comment]
live on prod since a while, all good

/ocabot merge minor

**Pull request merged**